docs/source/technical_tutorials/dps_tutorial/DPS_runner_template.ipynb

```diff
@@ -0,0 +1,353 @@
+{
```

Comment:
<p>this might be confusing to some as it's mixing shell/bash commands into Python cells. Could use the python equivalent.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/408/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>